submissions/hisbvdis/html-css-popup/index.html

```diff
@@ -0,0 +1,162 @@
+                  <span class="popup__title">Mail</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--news"></span>
+                  <span class="popup__title">News</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--maps"></span>
+                  <span class="popup__title">Map</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--files"></span>
+                  <span class="popup__title">Files</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--contacts"></span>
+                  <span class="popup__title">Contacts</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--translation"></span>
+                  <span class="popup__title">Translate</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--videos"></span>
+                  <span class="popup__title">Videos</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--search"></span>
+                  <span class="popup__title">Search</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--photo"></span>
+                  <span class="popup__title">Photo</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--apps"></span>
+                  <span class="popup__title">Apps</span>
+                </a>
+              </li>
+              <li class="popup__item">
+                <a class="popup__link" href="#">
+                  <span class="popup__icon  popup__icon--document"></span>
+                  <span class="popup__title">Document</span>
+                </a>
+              </li>
+            </ul>
+
+            <input class="popup__moreBtn  srOnly" id="popup__moreBtn" type="checkbox" />
+            <label class="popup__moreBtnLabel" for="popup__moreBtn">More</label>
```

Comment:
Add styles on focus state. To indicate to the user where the page focus is.

Reply:
+